include/crocoddyl/multibody/actions/impulse-fwddyn.hpp

```diff
@@ -8,61 +8,96 @@
 
 #ifndef CROCODDYL_MULTIBODY_ACTIONS_IMPULSE_FWDDYN_HPP_
 #define CROCODDYL_MULTIBODY_ACTIONS_IMPULSE_FWDDYN_HPP_
-
+#include "crocoddyl/multibody/fwd.hpp"
 #include <stdexcept>
+#include "crocoddyl/core/utils/exception.hpp"
 #include "crocoddyl/core/action-base.hpp"
 #include "crocoddyl/multibody/states/multibody.hpp"
 #include "crocoddyl/multibody/actuations/floating-base.hpp"
 #include "crocoddyl/multibody/impulses/multiple-impulses.hpp"
 #include "crocoddyl/multibody/data/impulses.hpp"
 #include "crocoddyl/multibody/costs/cost-sum.hpp"
+#include "crocoddyl/multibody/actions/impulse-fwddyn.hpp"
+
+#include <pinocchio/algorithm/compute-all-terms.hpp>
```

Comment:
Do we need to include these pinocchio stuff?
As far I understand, we should include them in the hxx file, right?

Reply:
If the answer is yes, then move the pinocchio headers up, i.e. include them before crocoddyl stuff but after stdexcept